docs/submitting_data.rst

```diff
@@ -168,7 +168,7 @@ data structures to send, do something like this.
 
 .. code-block:: python
 
-    from thclient import (TreeherderAuth, TreeherderClient, TreeherderClientError,
+    from thclient import (HawkAuth, TreeherderClient, TreeherderClientError,
```

Comment:
There was a good idea in the upstream discussion of just passing a tuple with the credentials, which I think we should do regardless. We could do this by checking the type inside TreeherderClient's constructor:

e.g.:

    if type(auth) == tuple:
       # create hawk auth object
    else:
       # fall back to legacy authentication strategies